Bucketeer/Sources/Internal/Remote/ApiClientImpl.swift

```diff
@@ -102,7 +103,7 @@ final class ApiClientImpl: ApiClient {
                 case .success((let response, _)):
                     completion?(.success(response))
                 case .failure(let error):
-                    completion?(.failure(.init(error: error)))
+                    completion?(.failure(.init(error: error).copyWith(timeoutMillis: timeoutMillisValue)))
```

Comment:
```suggestion
                    completion?(.failure(.init(error: error).copyWith(timeoutMillis: defaultRequestTimeoutMills)))
```

Reply:
changed in https://github.com/bucketeer-io/ios-client-sdk/pull/45/commits/430d9306f0552f9373d50548a9296fe93037eba8

Reply:
changed in https://github.com/bucketeer-io/ios-client-sdk/pull/45/commits/430d9306f0552f9373d50548a9296fe93037eba8